docs/installation.rst

```diff
@@ -78,20 +78,40 @@ Standard Installation
 
 3. Save ``generated_values.yaml``, somewhere safe. This is your Helm ``values.yaml`` file.
 
-4. Download the Helm chart and Install Robusta using Helm. On some clusters this can take a while [#f2]_, so don't panic if it appears stuck:
+.. admonition:: Multiple clusters
+    :class: important
+
+    Use the same ``generated_values.yaml`` for all your clusters (dev, prod, etc..). There's no need to run gen-config again.
+
+Standard Installation
+------------------------------
+
+1. Download the Helm chart:
 
 .. code-block:: bash
-   :name: cb-helm-install-robusta
+   :name: cb-helm-repo-add-update-robusta
 
     helm repo add robusta https://robusta-charts.storage.googleapis.com && helm repo update
-    helm install robusta robusta/robusta -f ./generated_values.yaml
 
-5. Verify that Robusta is running two pods and there are no errors in the logs:
+2. Copy the cluster name from the current context and install Robusta using Helm. On some clusters this can take a while [#f2]_, so don't panic if it appears stuck:
+
+.. admonition:: Test clusters (e.g Kind, MiniCube, Colima)
+    :class: important
+
+    Test clusters tend to have fewer resources. To lower the resource requests of Robusta,
+    include ``--set isSmallCluster=true`` at the end of the install command.
+
+.. code-block:: bash
+   :name: cb-helm-install-only-robusta
+
+    helm install robusta robusta/robusta -f ./generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME> # --set isSmallCluster=true
+
+3. Verify that Robusta is running at least two pods and there are no errors in the logs:
```

Comment:
Not sure about "running at pod"
Maybe:
Verify the two Robusta pods and running with no errors in the logs

Reply:
there are more pods for robusta if you install prometheus